book/src/json-output.md

```diff
@@ -68,6 +71,19 @@ Afterwards `levels.json` looks like this:
 {"data":"println","host_timestamp":1643113389707313290,"level":null,"location":{"file":"src/bin/levels.rs","line":15,"module_path":{"crate_name":"levels","modules":[],"function":"__cortex_m_rt_main"}},"target_timestamp":"4"}
 ```
 
+One use case is changing the human readable output with a piped command. 
```

Comment:
I would suggest moving this addition to the very end and mention that `jq` is an existing tool that can be used to change the output format. But again, the example output should be be JSON but something more readable.

Reply:
> But again, the example output should be be JSON but something more readable.

*should **not** be JSON*.

by "example output" I meant the code block. Ideally, it should be something like

``` console
$ cargo r --example foo | jq "something here"
I am a PRINTLN-statement
the next log statement
```

Reply:
> I would suggest moving this addition to the very end and mention that `jq` is an existing tool that can be used to change the output format. But again, the example output should be be JSON but something more readable.

Dumb one, do you mean at the very end of the page or the very end of the paragraph? I fear it will be a bit out of context at the very end of the page.

Reply:
I meant at the end of the page. After re-reading the page I think a good place may be after the first paragraph of the "Data transfer objects"

> So, what can I do with the JSON output?
> There really are no boundaries. (..)
> (NEW) One option is to use a program like `jq` to extract the parts of interest. *blah* *blah* *show console example*
> If you wish to deserialize (..)

Reply:
Maybe that sentence is enough. Everybody already saw a json output. The console example does not bring anything.